src/osdc/Objecter.cc

```diff
@@ -703,7 +703,7 @@ void Objecter::_send_linger_ping(LingerOp *info)
   logger->inc(l_osdc_linger_ping);
 }
 
-void Objecter::_linger_ping(LingerOp *info, int r, mono_time sent,
+void Objecter::_linger_ping(LingerOp *info, int r, ceph::coarse_mono_time sent,
 			    uint32_t register_gen)
```

Comment:
It looks like we don't actually use the sent parameter here. If so we should just get rid of it.